src/main/java/com/demo/classroom/Security/Service/JwtService.java

```diff
@@ -20,18 +20,21 @@
 import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.stereotype.Service;
 
+import com.demo.classroom.Utility.Constants.Expiration;
+
 @Service
 public class JwtService {
 
-    private final String SECRET_KEY;
-    private final long JWT_EXPIRATION;
-    private final long REFRESH_EXPIRATION_TIME;
+    private final String JWT_SECRET_KEY;
+    private final long ACCESS_TOKEN_EXPIRATION_TIME;
+    private final long REFRESH_TOKEN_EXPIRATION_TIME;
+    private long currentTimeMillis = System.currentTimeMillis();
 
     @Autowired
     public JwtService(Environment env) {
-        this.SECRET_KEY = env.getProperty("JWT_SECRET_KEY");
-        this.JWT_EXPIRATION = 900000; 
-        this.REFRESH_EXPIRATION_TIME=3600000;
+        this.JWT_SECRET_KEY = env.getProperty("JWT_SECRET_KEY");
+        this.ACCESS_TOKEN_EXPIRATION_TIME = Expiration.ACCESS_TOKEN_EXPIRATION_TIME.getValue(); 
+        this.REFRESH_TOKEN_EXPIRATION_TIME= Expiration.REFRESH_TOKEN_EXPIRATION_TIME.getValue();
```

Comment:
Can we rename it like this?
```suggestion
        this.JWT_SECRET_KEY = env.getProperty("JWT_SECRET_KEY");
        this.ACCESS_TOKEN_EXPIRATION_TIME = ExpirationTime.ACCESS_TOKEN.getValue(); 
        this.REFRESH_TOKEN_EXPIRATION_TIME= ExpirationTime.REFRESH_TOKEN.getValue();
```

Reply:
done